osu.Game/Overlays/Settings/Sections/Gameplay/AudioSettings.cs

```diff
@@ -2,27 +2,51 @@
 // See the LICENCE file in the repository root for full licence text.
 
 using osu.Framework.Allocation;
+using osu.Framework.Bindables;
 using osu.Framework.Graphics;
 using osu.Framework.Localisation;
 using osu.Game.Configuration;
 using osu.Game.Localisation;
+using osu.Framework.Graphics.Containers;
 
 namespace osu.Game.Overlays.Settings.Sections.Gameplay
 {
     public class AudioSettings : SettingsSubsection
     {
         protected override LocalisableString Header => GameplaySettingsStrings.AudioHeader;
 
+        private Bindable<float> positionalHitsoundsLevel;
+
+        private FillFlowContainer<SettingsSlider<float>> positionalHitsoundsSettings;
+
         [BackgroundDependencyLoader]
-        private void load(OsuConfigManager config)
+        private void load(OsuConfigManager config,OsuConfigManager osuConfig)
```

Comment:
Why duplicate two `OsuConfigManager`

Reply:
First of all it wouldn't kill you to be less rude. And the reason I duplicated it was because I was unaware of how to direct osuconfigmanager to config and osuconfig at once.

Reply:
the name of the injected variable doesn't matter. it's fine to just use `config` everywhere, there is no difference between the two